### PR TITLE
Bug 1138278 - [Clock] After running test_clock_create_new_alarm.py, it s...

### PR DESCRIPTION
**Pull request by alison-shiue**
Fixed this issue based on Comment 4 in Bug 1138278. (According to bug 1128952, refactor time_picker.py.)
@JohanLorenzo could you please help review it? thanks.

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7e42eb9e4a8719a2b1a5382894da511243c26f47

File: tests/python/gaia-ui-tests/gaiatest/apps/system/regions/time_picker.py [hunk 1]
```diff
@@ -16,54 +17,44 @@ class TimePicker(Base):
...
+        # TODO: wait for the time picker to fade out Bug 1038186
         time.sleep(2)
         self.apps.switch_to_displayed_app()
 
     @property
     def hour(self):
-        return self.marionette.find_element(*self._current_element(*self._hour_picker_locator)).text
-
+        return self._current(self._hour_picker_locator).text
+
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/system/regions/time_picker.py]: Nit: Trailing whitespace

File: tests/python/gaia-ui-tests/gaiatest/apps/system/regions/time_picker.py [hunk 2]
```diff
@@ -16,54 +17,44 @@ class TimePicker(Base):
...
     @property
     def hour(self):
-        return self.marionette.find_element(*self._current_element(*self._hour_picker_locator)).text
-
+        return self._current(self._hour_picker_locator).text
+         
     @property
     def minute(self):
-        return self.marionette.find_element(*self._current_element(*self._minutes_picker_locator)).text
+       return self._current(self._minutes_picker_locator).text
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/system/regions/time_picker.py]: Nit: The indentation is not a multiple of 4 anymore.

File: tests/python/gaia-ui-tests/gaiatest/apps/system/regions/time_picker.py [hunk 3]
```diff
@@ -16,54 +17,44 @@ class TimePicker(Base):
...
 
     def spin_minute(self):
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/system/regions/time_picker.py]: For consistency with the master branch, I would rename this function `add_minute` like [done here](https://github.com/mozilla-b2g/gaia/commit/59f71e42d4137f194ca50709497f6e3706dad10a#diff-3ec38c276967a961e2f4c4ac0c8fe0f9R53)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=90386980bb3dedda97f938188bc425bbda3762b0

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=90386980bb3dedda97f938188bc425bbda3762b0

**Comment by JohanLorenzo** [comment]
I found this PR thanks to https://github.com/pulls/mentioned. There hasn't been any activity for more than 3 years. I'm closing it for the sake of cleanup (Gaia is not a supported project anymore). If you think it still has some use, please reopen it.

**Pull request closed without merging**